### PR TITLE
Resource Management cleanup

### LINKED ISSUE #356: Text Clean-up: Section 3 - Resource Management
This issue is to update the text in Section 3 per the comments in the [this Google-Doc](https://docs.google.com/document/d/1n7ZVIRaiYG36Ye0AP4oSTYbDN915box6zVteXciojFc/edit?usp=sharing).

The updates should also include:
1. A `3.1 Introduction`
1. Replacement of "An Implementation" with "Implementations"... as applicable.

### PR DESCRIPTION
**Pull request by escowles**
Fixes #356

### CODE REVIEW

**Review by zimeon — DISMISSED** [review]
Very much like creation of constraints document section!

File: index.html [hunk 1]
```diff
@@ -290,16 +290,26 @@ <h2>LDP-NR creation</h2>
...
             Non-normative note: Neither [[LDP]] nor this specification constrains the underlying storage model for RDF
             data. Each <a>LDP-RS</a> may be entirely independent RDF content, may be a portion of a shared RDF store,
-            or follow some other arrangement. Implementation choices that result in failure to complete client requests
-            are expressed in a constraints document ([[LDP]]
-            <a href="https://www.w3.org/TR/ldp/#ldpr-gen-pubclireqs">4.2.1.6</a>) as noted in the appropriate sections
-            of [[LDP]] and below.
+            or follow some other arrangement.
           </blockquote>
+          <p>
+            If a client request cannot processed because of data model constraints, the response MUST include an
```
Comment (dannylamb) [inline index.html]: s/cannot processed/cannot be processed

File: index.html [hunk 2]
```diff
@@ -322,7 +332,7 @@ <h2>Additional values for the <code>Prefer</code> header</h2>
             </li>
             <li>
               Implementations MAY support <code>http://www.w3.org/ns/oa#PreferContainedDescriptions</code>
-              to request that the response include representations of any <a>LDPC</a> contained resources in the
+              to request that the response include representations of any <a>LDPC</a>-contained resources in the
```
Comment (dannylamb) [inline index.html:335]: s/include/includes

File: index.html [hunk 3]
```diff
@@ -395,18 +406,16 @@ <h2>HTTP OPTIONS</h2>
...
         <p>
-          Any <a>LDPC</a> (except <a href="#ldpcv-post">Version Containers (LDPCv)</a>) MUST support <code>POST</code>
+          Any <a>LDPC</a> except <a href="#ldpcv-post">Version Containers (LDPCv)</a> MUST support <code>POST</code>
           ([[!LDP]] <a href="https://www.w3.org/TR/ldp/#ldpr-HTTP_POST">4.2.3</a> /
           <a href="https://www.w3.org/TR/ldp/#ldpc-HTTP_POST">5.2.3</a>). The default interaction model that will be
-          assigned when one is not specified in a <code>Link</code> header in the request MUST be recorded in the
-          constraints document referenced in the <code>rel="http://www.w3.org/ns/ldp#constrainedBy"</code> link in
-          the <code>Link</code> response header ([[!LDP]]
-          <a href='https://www.w3.org/TR/ldp/#ldpr-gen-pubclireqs'>4.2.1.6</a> clarification).
+          assigned when one is not specified in a <a href="#constraints-document">constraints document</a>.
```
Comment (dannylamb) [inline index.html]: assinged when one is not specified _must be recorded_ in a constraints document

**Review by dannylamb — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (awoods) [reply index.html:335]: I am not sure about this grammar suggestion, @dannylamb .

**Review by awoods — CHANGES_REQUESTED** [review]
Pending minor grammatical suggestions from @dannylamb .

File: index.html (hunk 2 above)
Comment (escowles) [reply index.html:335]: I like "include" better here — though maybe it should be "that the response should include"?

**Review by awoods — APPROVED** [review]

**Review by zimeon — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (dannylamb) [reply index.html:335]: Sure enough, it's `he/she/it includes`, but `he/she/it include` if it's subjunctive, which I guess this is.  I stand corrected.

**Review by dannylamb — APPROVED** [review]

**Review by barmintor — APPROVED** [review]

**Pull request merged**